tests/testthat/test-write.R

```diff
@@ -4,6 +4,9 @@ data_to_save <- function() {
     as_tibble()
 }
 
+# Skip large file tests unless explicitly requested
```

Comment:
When does this run?  Manually by us?

Reply:
Yeah, This would melt any pipeline and most laptops so its only should be run on a beefier machine when you need to run it.